xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -561,6 +561,9 @@ void CGUIDialogPVRTimerSettings::OnSettingAction(const std::shared_ptr<const CSe
 
 bool CGUIDialogPVRTimerSettings::Validate()
 {
+  if (m_timerType->IsTimerRule()) // no start/end day spinners, so no point in checking time
+    return true;
```

Comment:
I'm not sure we can do this here, I believe we can have rules that also have day spinners

Reply:
In theory yes but are there any backends actually supporting this? Eg. TVHeadend doesn't afaik....